packages/text-input-react/documentation/Autosuggest.mdx

```diff
@@ -19,14 +19,8 @@ import AutosuggestExample, { autosuggestExampleCode } from "./AutosuggestExample
     <a className="jkl-link jkl-link--external" href="https://github.com/downshift-js/downshift">
         downshift.js
     </a>{" "}
-    isteden for{" "}
-    <a
-        className="jkl-link jkl-link--external"
-        href="https://static.nrk.no/core-components/latest/index.html?core-suggest/readme.md"
-    >
-        NRK Core-components
-    </a>
-    . Du må gjerne bruke denne, men forvent at Autosuggest kommer til å bli skrevet om. Det kommer til å være breaking changes.
+    . Du må gjerne bruke denne, men forvent at Autosuggest kommer til å bli skrevet om. Det kommer til å være breaking
```

Comment:
Nå har denne vært merket eksperimentell ganske lenge. På tide å forfremme den?

Reply:
Jeg har egentlig lyst til å demote den ut av Jøkul. Er det noen som bruker den, i så fall til hva og hvordan skiller det seg fra select med søk?

Reply:
Godt spørsmål. Litt vanskelig å si noe om bruk med innsikten vi har tilgjengelig nå (siden den er i text-input pakka; ingen egen download-counter). Må nesten bare ta en spørrerunde. Men godt poeng at denne og søkbar Select har en del overlapp.

Reply:
Den brukes i noen av løsningene våre (usikker på om jeg skal skrive hvilke her på GitHub). Den skiller seg fra Select med søk ved at man kan skrive inn en verdi som ikke finnes i lista. Det er for å dekke situasjoner der vi ønsker å komme med forslag fra våre egne kategorier, men uten å begrense brukeren helt, for eksempel hvis man spør om yrke.

Reply:
Er for øvrig enig med Jo Emil om at den heller bør ut enn inn, men da må vi sørge for å få dekket funksjonaliteten som forsvinner.

Reply:
Det er en nyanseforskjell der, det med å faktisk kunne velge noe som ikke er en `<option>`. Jeg vil ikke blande inn det konseptet i en Select.

Men synes da at den snart må kunne sies å være en ✨blessed ✨ del av Jøkul. Synes kanskje den bør få sin egen pakke også.

Reply:
Uansett, litt off topic for PRen sånn egentlig – my bad 😄 Kan ta det opp i en egen tråd.